modules/uncertainty/R/ensemble.R

```diff
@@ -207,28 +207,28 @@ write.ensemble.configs <- function(defaults, ensemble.samples, settings, model,
     return(list(runs = NULL, ensemble.id = NULL))
   }
   
+  # See if we need to write to DB
+  write.to.db <- as.logical(settings$database$bety$write)
   # Open connection to database so we can store all run/ensemble information
-  if (write.to.db) {
```

Comment:
I don't understand why this was rewritten to drop this `if(write.to.db)` and to instead attempt to open that connection regardless.

Reply:
Indeed, I would think we still want a check to see if the user has selected to write or not, if not it simplifies this.  I dont think we want to assume that we want write to bety

Reply:
So @serbinsh and I seem to agree that we want to retain this check, but disagree about the default -- I *DO* think we want to assume that we want to write to bety by default.

Reply:
The default behavior is set by the function's argument which will be `TRUE` to write to bety.